mp/src/game/server/hl2mp/hl2mp_player.cpp

```diff
@@ -608,6 +608,13 @@ void CHL2MP_Player::FireBullets ( const FireBulletsInfo_t &info )
 	if ( pWeapon )
 	{
 		modinfo.m_iPlayerDamage = modinfo.m_flDamage = pWeapon->GetHL2MPWpnData().m_iPlayerDamage;
+#ifdef NEO
+		if (modinfo.m_iAmmoType == 13)
+		{
+			// supa7 slug does about this much damage more than a single shot pellet
```

Comment:
Afaik, the slug has the same damage as Kyla

Reply:
which actually is 18*2.5